command/pr_checkout.go

```diff
@@ -27,10 +28,11 @@ func prCheckout(cmd *cobra.Command, args []string) error {
 	}
 
 	var baseRepo ghrepo.Interface
-	prArg := args[0]
-	if prNum, repo := prFromURL(prArg); repo != nil {
-		prArg = prNum
-		baseRepo = repo
+	prString := args[0]
+	r := regexp.MustCompile(`^https://github\.com/([^/]+)/([^/]+)/pull/(\d+)`)
+	if m := r.FindStringSubmatch(prString); m != nil {
+		prString = m[3]
+		baseRepo = ghrepo.New(m[1], m[2])
```

Comment:
This respects the base repo as found in the URL. 👍  Any reason this is currently only scoped to `pr checkout` and not extended to all pr commands via `pr_lookup.go`?